README.md

```diff
@@ -93,6 +93,42 @@ Fill out [config](./config.yaml) file with your details (or leave default values
 | *PASSWORD*            | *null* **(need to be overwritten)**   | User's password
 | *INTERACTIVE_MODE*    | *True*                                | Enables interactive mode (confirmation prompts).
 
+---
+### Alexa skill configuration (*Optional*)
+
+You can configure an Alexa skill to control the Sidewalk device via voice commands. This is however optional and you can skip this step if you don't want to use Alexa with this sample app. 
+
+Install the required libraries for Alexa skill.
+
+- Linux / MacOS:
+```
+python3 -m pip install requests  -t .\ApplicationServerDeployment\lambda\alexaLibs
+python3 -m pip install 'urllib3<2'  -t .\ApplicationServerDeployment\lambda\alexaLibs --upgrade
+```
+
+- Windows:
+```
+python -m pip install requests  -t .\ApplicationServerDeployment\lambda\alexaLibs
+python -m pip install 'urllib3<2'  -t .\ApplicationServerDeployment\lambda\alexaLibs --upgrade
+```
+
+| Follow the guide below to create an Alexa skill (*Optional) |
```

Comment:
```suggestion
| Follow the guide below to create an Alexa skill |
```
Remove 'Optional' as this is already mentioned in the section topic.

Reply:
Agreed with this change

Reply:
Agreed with this change